app/hotels/src/singleHotel/roomList/BeddingInfo.js

```diff
@@ -19,16 +19,29 @@ type Props = {
 };
 
 export class BeddingInfo extends React.Component<Props> {
-  formatBeddingInfo = (): string => {
+  formatBeddingInfo = (): $ReadOnlyArray<React.Element<typeof Translation>> => {
     const { room } = this.props;
     const beddingOptions = room?.bedding ?? [];
-    return beddingOptions
-      .map(beddingOption => {
-        const type = beddingOption?.type;
-        const amount = beddingOption?.amount;
-        return [amount, type].filter(value => value != null).join(' ');
-      })
-      .join(' or '); // TODO: Translate
+    const beddingTranslation = [];
+    beddingOptions.forEach((beddingOption, index) => {
+      const type = beddingOption?.type;
+      const amount = beddingOption?.amount;
+      if (index > 0) {
+        beddingTranslation.push(
+          <Translation
+            key={`key-${index}`}
+            id="single_hotel.bedding_info.or"
+          />,
+        );
+      }
+      beddingTranslation.push(
+        <Translation
+          key={`${type || ''}-${amount || ''}`}
+          passThrough={[amount, type].filter(value => value != null).join(' ')}
```

Comment:
I don't know if that always works for any language. But I guess that's good enough for now?

Reply:
Amount and type is being returned translated from booking.com based on the language parameter, so this should be fine.

Reply:
Sure, but I meant the order of the words, maybe in some languages you would have `bananas 2` and not `2 bananas` like we are used to.